source/server/player/player_core.qc

```diff
@@ -592,6 +592,9 @@ void() PlayerPostThink =
 
 	// Network everything
 	self.SendFlags = 1;
+
+	// Used to tell which gun of the mustang and sally to shoot
```

Comment:
Please revise this comment, there are more akimbo weapons than just the Mustang & Sally, and there should be further context here as to why this is relevant.